src/test/java/ladder/domain/participant/ParticipantsTest.java

```diff
@@ -36,7 +42,50 @@ void checkParticipantsCountTest(final List<String> names) {
                 .isInstanceOf(InvalidParticipantsCountException.class);
     }
 
-    static Stream<List<String>> getInvalidParticipantsNames() {
+    @Test
+    @DisplayName("이름 순서에 맞춰 참가자에게 Position을 부여한다.")
+    void assignPositionTest() {
+        // given
+        final List<String> names = List.of("poo", "po", "p");
+        final List<Position> expectedPositions = IntStream.range(0, 3)
+                .mapToObj(Position::new)
+                .toList();
+
+        // when
+        final List<Participant>  participants = new Participants(names).getValues();
```

Comment:
```suggestion
        final List<Participant> participants = new Participants(names).getValues();
```

[의견] 커밋 전에(또는 자동으로) 컨벤션에 맞게 포맷팅도 꼭 해주시면 좋아요 :)